### ISSUE TITLE
Python PlugIn doesn't Work Ubuntu

### ISSUE THREAD

**Issue by eapuertoc** [NREL/EnergyPlus#8763]
Issue overview
--------------
When using the same python plugin created for Windows, doesn't work in Ubuntu, throws the following error:
File "/home/user/Documents/Workspace/energyplus/EvapMistCooling.py", line 1, in <module>
    from pyenergyplus.plugin import EnergyPlusPlugin
   **   ~~~   **  >>>   File "/usr/local/EnergyPlus/pyenergyplus/plugin.py", line 3, in <module>
    from pyenergyplus.api import EnergyPlusAPI
   **   ~~~   **  >>>   File "/usr/local/EnergyPlus/pyenergyplus/api.py", line 1, in <module>
    from ctypes import cdll, c_char_p, c_void_p
   **   ~~~   **  >>>   File "/usr/local/EnergyPlus/python_standard_lib/ctypes/__init__.py", line 7, in <module>
    from _ctypes import Union, Structure, Array
   **   ~~~   **  >>> ImportError: libffi.so.6: cannot open shared object file: No such file or directory
   **   ~~~   ** ```

### Details
 - Ubuntu 20.04
 - Energyplus 9.5

**Comment by bonnema** [comment]
Can you share the .py file?

**Comment by Myoldmopar** [comment]
I believe on my 20.04 machine I have run into this exact same message when I accidentally grabbed the 18.04 installer.  Are you sure you got the 20.04 installer package and not the 18.04?  Just wanting to eliminate any easy fixes first...but yes, if you are definitely running the 20.04 installer, then send along your input file, weather file if appropriate, and definitely the python file.

**Comment by eapuertoc** [comment]
Hi everybody. Unfortunately I can't upload the exact file because is part of a confidential project in my university, but I share a [simplified model with the same issue](https://github.com/eapuertoc/UbuntuTest). About the version, using #lsb_release -a in my terminal I have the following result:

No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 20.04.2 LTS
Release:	20.04
Codename:	focal

I don't think is a idf or python module problem because the same code works on Windows.

**Comment by bonnema** [comment]
`lsb_release` is for the Ubuntu operating system version. Can you share the output of the `/usr/local/EnergyPlus-9-5-0/energyplus --help` command? I would bet that @Myoldmopar is (as usual) correct, the default link on energyplus.net is for the 18.04 installer. Try one of the 20.04 installers here: https://github.com/NREL/EnergyPlus/releases/tag/v9.5.0.

**Comment by Myoldmopar** [comment]
> @Myoldmopar is (as usual) correct

New tagline, thanks @bonnema !

@eapuertoc thanks for the updated info, but like @bonnema was saying there, we understand you are using Ubuntu 20.04, I think you just got the installer for 18.04 by mistake.  I'm not actually sure the output of `--help` will help here because it doesn't mention the target build platform.  Please try to download the 20.04 installer from the link that @bonnema provided and let us know if that help.

Whether this exactly fixes the problem or not, I have a couple todo's here:
- Add another link on the download page that points to either Ubuntu 18.04 or Ubuntu 20.04.
- Add an output line on `--help` that describes the target (or build) platform, at least on Ubuntu.

**Comment by bonnema** [comment]
@Myoldmopar you are correct (again😀) that `--help` won't actually, well, help. My bad.

Would a quick fix be to add the OS version to the buttons on https://energyplus.net/downloads? Then at least the user would be aware of what they're getting? Might apply to Mac as well (10.15 v. 11.2). Or have smaller links below like Windows has for 32-bit for the less-used version for Ubuntu and Mac?

Like:
**Ubuntu (18.04)** **Mac (10.15)**
Ubuntu (20.04) Mac (11.2)

**Comment by Myoldmopar** [comment]
Yeah, that's what we should do on the website.  And for future releases, we'll get the `--help` output modified to report the target version, again at least on Ubuntu.

**Comment by eapuertoc** [comment]
OK, now I understand. I just downloaded the 20.04 version in repository and works. My apologies, I download the version direct in the page and didn't notice the name of the install file said "1804". Thank you for you support.

**Comment by Myoldmopar** [comment]
OK, I am tackling the `--help` issue now.  In a local branch I now have it able to report out the platform that the package was built on:

```sh
$ ./energyplus --help
EnergyPlus, Version 9.5.0-32c2638e17, YMD=2021.05.20 12:32
PythonLinkage: Linked to Python Version: "3.8.5 (default, Jan 27 2021, 15:41:15) [GCC 9.3.0]"
Built on Platform: Ubuntu20.04_x86_64
Usage: energyplus [options] [input-file]
Options:
  -a, --annual                 Force annual simulation
```

That will help diagnose when something goes wrong.  I have also submitted a request to tweak the Downloads page to indicate the Linux packages are available for **either** 18.04 or 20.04, which should help users get the correct version.

Thanks to those on here helping get the issues understood and addressed.

**Comment by Myoldmopar** [comment]
Download page updated: https://energyplus.net/downloads:

![image](https://user-images.githubusercontent.com/849824/119388152-843f0580-bc8f-11eb-9462-0c33afcb2901.png)
